resource-definitions/terraform-driver/runner-pod-configuration/k8s-cluster-refs.tf

```
@@ -0,0 +1,30 @@
+resource "humanitec_resource_definition" "aks_resource_cluster" {
+  id          = "eks-enabled-cluster"
+  name        = "eks-enabled-cluster"
+  type        = "k8s-cluster"
+  driver_type = "humanitec/k8s-cluster-aks"
+
+  driver_inputs = {
+    secrets = {
+      credentials = {
+        appId       = var.app_id
+        displayName = var.display_name
+        password    = var.password
+        tenant      = var.tenant
+      }
+    }
+
+    values = {
+      name            = "my-cluster"
+      resource_group  = "my-azure-resource-group"
+      subscription_id = "123456-1234-1234-1234-123456789"
+      server_app_id   = "6dae42f8-4368-4678-94ff-3960e28e3630"
+    }
+  }
+}
+
+
+resource "humanitec_resource_definition_criteria" "aks_aad_resource_cluster" {
+  resource_definition_id = humanitec_resource_definition.aks_aad_resource_cluster.id
```

```suggestion
  resource_definition_id = humanitec_resource_definition.aks_resource_cluster.id
```
Align resource name with the humanitec_resource_definition object above. Alternatively, rename the cluster "humanitec_resource_definition" to "aks_aad_resource_cluster".